### PR TITLE
Update Jasper eval

### PR DESCRIPTION
**Pull request by blisc**
update Jasper Eval
switch nemo.logging to logging

### CODE REVIEW

**Comment by vsl9** [comment]
Can you please update `examples/nlp/language_modeling/bert_pretraining.py` to resolve the merge conflict?

File: examples/asr/jasper_eval.py [hunk 1]
```diff
@@ -132,13 +143,28 @@ def main():
     eval_tensors = [log_probs_e1, predictions_e1, transcript_e1, transcript_len_e1, encoded_len_e1]
 
     # inference
-    evaluated_tensors = neural_factory.infer(tensors=eval_tensors, checkpoint_dir=load_dir, cache=False)
+    cache = bool(args.lm_path)
+    if cache:
+        logging.warning(
+            "Caching is not recommended for large datasets as they might not fit in cpu memory. To run beam search "
+            "decoding, it is recommended to dump log_probalities to disk using the --save_logprob argument."
```
Comment (vsl9) [inline examples/asr/jasper_eval.py]: Can you please fix the typo?

File: nemo/collections/asr/beam_search_decoder.py [hunk 2]
```diff
@@ -59,7 +59,9 @@ def output_ports(self):
         # return {"predictions": NeuralType(VoidType())}
         return {"predictions": NeuralType(('B', 'T'), PredictionsType())}
 
-    def __init__(self, vocab, beam_width, alpha, beta, lm_path, num_cpus, cutoff_prob=1.0, cutoff_top_n=40):
+    def __init__(
+        self, vocab, beam_width, alpha, beta, lm_path, num_cpus, cutoff_prob=1.0, cutoff_top_n=40, input_tensor=True
```
Comment (vsl9) [inline nemo/collections/asr/beam_search_decoder.py]: Can you please add a line describing `input_tensor` parameter into the class docstring?

Comment (blisc) [reply nemo/collections/asr/beam_search_decoder.py]: done

File: examples/asr/jasper_eval.py (hunk 1 above)
Comment (blisc) [reply examples/asr/jasper_eval.py]: msg removed

**Pull request merged**